tests/CMakeLists.txt

```diff
@@ -23,9 +23,14 @@ if(BUILD_GENIVI)
     list(APPEND TEST_LIBS rvi)
 endif(BUILD_GENIVI)
 
+add_custom_target(prepare-ostree-sysroot
+    COMMAND if [ ! -d "sysroot" ]; then curl https://s3.eu-central-1.amazonaws.com/pw-dropbox/ostree-sysroot.tar.gz -o ostree-sysroot.tar.gz && tar -xf ostree-sysroot.tar.gz \; fi
+    DEPENDS aktualizr)
 add_custom_target(build_tests DEPENDS aktualizr)
-add_custom_target(check COMMAND CTEST_OUTPUT_ON_FAILURE=1 ${CMAKE_CTEST_COMMAND} -E test_valgrind_uptane_vectors\\|test_build DEPENDS build_tests)
-add_custom_target(check-full COMMAND CTEST_OUTPUT_ON_FAILURE=1 ${CMAKE_CTEST_COMMAND} -E test_uptane_vectors DEPENDS build_tests)
+add_custom_target(build_tests_ci DEPENDS prepare-ostree-sysroot)
+add_custom_target(check COMMAND CTEST_OUTPUT_ON_FAILURE=1 ${CMAKE_CTEST_COMMAND} -E test_valgrind_uptane_vectors\\|test_build\\|test_uptane_ci DEPENDS build_tests)
+add_custom_target(check-full COMMAND CTEST_OUTPUT_ON_FAILURE=1 ${CMAKE_CTEST_COMMAND} -E test_uptane_vectors\\|test_uptane_ci DEPENDS build_tests)
+add_custom_target(check-ci COMMAND CTEST_OUTPUT_ON_FAILURE=1 ${CMAKE_CTEST_COMMAND} -R test_uptane_ci DEPENDS build_tests_ci)
```

Comment:
Would it be simpler to enable this test only when SOTA_PACKED_CREDENTIALS is set, and skip it otherwise?